### PR TITLE
Travis: Turn on linting

### PR DESCRIPTION
**Pull request by keradus**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -5,12 +5,13 @@ matrix:
...
         - php: 5.3
           env: 'COMPOSER_FLAGS="--prefer-stable --prefer-lowest"'
-        - php: 7.0
         - php: hhvm
         - php: 5.4
         - php: 5.5
         - php: 5.6
         - php: 7.0
+          env: LINT_TEST_CASES=1
+        - php: 7.0
```
Comment (gharlan) [inline .travis.yml]: why? I would start it as early as possible.

Comment (gharlan) [inline .travis.yml]: I would start with the special ones (lowest deps, highest deps with linting, hhvm).
Then the regular ones (5.4, 5.5, 5.6).
Then the additional check for php 7 (with sf 2.8).

**Comment by keradus** [comment]
note: Integration tests may not be done yet

**Comment by keradus** [comment]
looks green ;)

File: Symfony/CS/Tests/AbstractIntegrationTest.php [hunk 2]
```diff
@@ -223,4 +225,35 @@ function (FixerInterface $fixer) {
...
+    {
+        if (!getenv('LINT_TEST_CASES')) {
+            return;
+        }
+
+        if ($this->isLintException($source)) {
+            return;
+        }
+
+        // On 2.x replace with:
```
Comment (GrahamCampbell) [inline Symfony/CS/Tests/AbstractIntegrationTest.php]: :-1: for including this comment in the code

Comment (keradus) [inline Symfony/CS/Tests/AbstractIntegrationTest.php]: just a little help for me:
```php


        // On 2.x replace with:
        // try {
        //     static::$linter->lintSource($source);
        // } catch (\Exception $e) {
        //     return $e->getMessage()."\n\nSource:\n$source";
        // }

File: Symfony/CS/Tests/AbstractIntegrationTest.php [hunk 3]
```diff
@@ -223,4 +225,28 @@ function (FixerInterface $fixer) {
                 ->setInputCode(null)
         );
     }
+
+    /**
+     * @return string|null
+     */
+    protected function lintSource($source)
+    {
+        if (!getenv('LINT_TEST_CASES')) {
```
Comment (SpacePossum) [inline Symfony/CS/Tests/AbstractIntegrationTest.php]: why check the `env` (again) and not if the linter `!==null` (which is what method really needs to know rigth?)

Comment (keradus) [inline Symfony/CS/Tests/AbstractIntegrationTest.php]: :+1:

On the beginning i don't touch `setUp...` ;)

**Comment by SpacePossum** [comment]
:+1:  LGTM  :)

**Pull request closed without merging**